host-interaction/process/inject/use-dll-injection.yml

```diff
@@ -0,0 +1,34 @@
+rule:
+  meta:
+    name: use dll injection
```

Comment:
propose: `inject DLL`

Reply:
Fixed in aa94a33. Also added _a_ as suggested by @williballenthin for the PE injection rule.